ember/app/templates/components/click-catcher.hbs

```diff
@@ -0,0 +1,19 @@
+<div
+  class="click-catcher {{if openTerm 'active'}}"
+  onclick={{action 'catchClick'}}>
+  <div
+    class="definition {{orientation}}"
+    style="top: {{top}}; left: {{left}}; bottom: {{bottom}};"
+  >
+    <div class="header">
+      <span>{{label}}</span>
```

Comment:
Do we need a `<span>` here?

Reply:
I think it's good practice not to leave text floating.

Reply:
It's not floating, its surrounded by `.header`